### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by TheEhsanSarshar**

### CODE REVIEW

File: readme.md
Comment (dai-shi) [inline readme.md:23]: we prefer to slim readme as much as possible. do you have an idea to make it compact? how about this?
```bash
npm install zustand # or yarn add zustand
```

Comment (TheEhsanSarshar) [reply readme.md:23]: Ok, I will do that.
but still, the readme is too heavy.
we should have a better doc

File: readme.md
Comment (dai-shi) [inline readme.md:20]:
```suggestion

## First create a store
```

**Review by dai-shi — APPROVED** [review]
The change is okay. Let's wait for fixing tests before merge.

**Pull request merged**
